src/en/proposals/revolution-rework.md

```diff
@@ -0,0 +1,166 @@
+- The **ability is one-time** and there is no way to remove this bandage once activated.
+- To know the player's affiliation before activation requires examining the character - it will answer it in examine window (being a revolutionary).
+- Once alerted _(1.6)_, all armbands automatically appear on the shoulders of the rebels.
+
+**FOR WHAT:** To get rid of stealthy revolutions, add "rebel entourage" and replace trivial meta icons.
+
+### 1.3 Mindshield
+Has no effect on revolutionaries, but prevents further recruitment _(1.8)_.
+
+### 1.4 Goldstein Books 
+- There is a **free** item in headrev uplink _(1.1)_ - a **“brainwash book”.**
+- Initially there is no books in the uplink _(1.1)_, but for every **[headrev count * 2]** revolutionary, the copy will be replenished by one piece. **[Maximum books = 6]**.
+- When read **(by pressing Z)**, the character becomes a revolutionary. If a loyalist or rebel does it, nothing happens.
+>Hereinafter, a loyalist is a character with a mindshield implant. In the points system, a loyalist and a head are different people.
+
+**FOR WHAT:** Diversify the method of recruitment, allow simple revolutionaries to recruit, and make a reference to 1984.
+
+### 1.5 Rebel Flag
+- There is another free item in headrev uplink _(1.1)_ - rebel flag, which **can be placed** in different departments.
+- Used exclusively for the points system _(3.2)_.
+- If the department contains **[number of loyalists / 5]**, it is **automatically removed** after a minute.
+- Normal crew can remove it manually activating the process for 1 minute (can't move). Can't be moved.
+- **Can't be staged after** the evacuation shuttle has been dispatched.
+
+**FOR WHAT:** to get rid of stealthy revolutions, diversify the strategic part of the mode, and supplement the points system.
+
+### 1.6 Alert
+> Automatic station notification:
+“Attention! According to automatic internal sensors, there is crew misconduct on the station. There is a threat of a full-scale mutiny. 
+We ask the station heads to take measures according to the SOP and common sense. If necessary, send a fax to the CC regarding the current situation.”
+
+#### This announcement comes after the fact:
+- **over 40%** of the crew have been turned to the revolutionary side 
+- **over 80%** of the rebels have exposed their holographic armband
+- **more than 2 heads or 50%** of the loyal crew are in a neutralized state within 5 minutes
+- **more than 30%** of the crew has dangerous items (homemade, contraband, explosives) in their inventory
+- **there are 3 or more rebel flags** on the station
+
+**One minute before the announcement itself, a message will be given to all headrevs that they have been revealed.**
+
+**FOR WHAT:** to get rid of covert revolutions. 
+
+### 1.7 Headrev
+**When a headrev dies**, revolutionaries are **not deconverted**, but there will be no new headrev. Calculation of the number of hedrevs is the same as before.
```

Comment:
I like this, it moves sec/crew's objective from killing headrev to finding all the revs.